variation-commons-core/src/main/java/uk/ac/ebi/eva/commons/core/models/VariantClassifier.java

```diff
@@ -23,7 +23,13 @@
  */
 public class VariantClassifier {
 
-    private static Pattern alphaRegExPattern = Pattern.compile("[A-Z]+");
+    private static final Pattern alphaRegExPattern = Pattern.compile("[A-Z]+");
+
+    private static final Pattern ACGTPattern = Pattern.compile("[ACGT]+");
+
+    private static final Pattern tandemRepeatsPattern = Pattern.compile("^\\([ACGT]+\\)\\d+$");
+
+    private static final Pattern tandemRepeatsExpandedPattern = Pattern.compile("^A{3,}$|^C{3,}$|^G{3,}$|^T{3,}$");
```

Comment:
I don't think we want to classify tandem repeat differently from the Insertion or deletion plus these two patterns would not catch expanded tandem repeat 
`TACTACTACTAC --> TACTAC` would classify as an indel but
`TAC(4) --> TAC(2)` would classify as a Tandem repeat